### PR TITLE
Include nodejs seed with connection to postgres, docs improvements.

### PR DESCRIPTION
**Pull request by dimeloper**
# Title

Include nodejs seed with connection to postgres, docs improvements

## Description

- Includes a new nodejs seed that offers connectivity to postgres databases
- Introduces a new section in README files to offer a better overview in regards to environment variables
- Last but not least, it introduces eslint, to enforce coding style across all nodejs seeds

## Related Issues

[List any related issues or pull requests that this pull request addresses]

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots (if applicable)

[Include any relevant screenshots to help explain the changes made by this pull request]

## Checklist

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have tested my changes thoroughly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Please add an `x` inside the square brackets to check off each item in the checklist.

## Additional Notes (if applicable)

[Include any additional notes or context that might be helpful for reviewers]

### CODE REVIEW

File: backend/node-js-express/node-js-express-postgresql/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- `POSTGRES_PASSWORD` = The password of the PostgreSQL user.
+- `POSTGRES_PORT` = The port that the PostgreSQL instance listens to. Most of the time defaults to 5432
+- `POSTGRES_SSL` = Set this to true if your PostgreSQL instance offers an encrypted connection (SSL)
+
+## Dependencies
+
+This seed uses:
+
+- [Express.js](https://expressjs.com/): A fast, minimalist web framework for Node.js.
+- [Mongoose](https://mongoosejs.com/): Elegant MongoDB object modeling for Node.js.
```
Comment (dkargatzis) [inline backend/node-js-express/node-js-express-postgresql/README.md]: This should be Postgres, right?

File: backend/node-js-express/node-js-express-postgresql/app/app.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    // Middleware for parsing JSON bodies
+    this.app.use(express.json());
+
+    // add routes
+    this.routes();
+  }
+
+  routes() {
+    this.app.use('/', RootController);
+    this.app.use('/health', HealthController);
```
Comment (dkargatzis) [inline backend/node-js-express/node-js-express-postgresql/app/app.js]: This endpoint should be `healthz` as per the health checks path in helm templates.

File: backend/node-js-express/node-js-express-postgresql/README.md (hunk 1 above)
Comment (dimeloper) [reply backend/node-js-express/node-js-express-postgresql/README.md]: In a way yes, I included all the major dependencies and a brief description.

File: backend/node-js-express/node-js-express-postgresql/app/app.js (hunk 2 above)
Comment (dimeloper) [reply backend/node-js-express/node-js-express-postgresql/app/app.js]: Ah ok, makes sense, thanks!

**Review by dkargatzis — APPROVED** [review]

**Pull request merged**